src/Farmer/Builders/Builders.Cosmos.fs

```diff
@@ -73,6 +81,7 @@ type CosmosDbContainerBuilder() =
         { Name = ResourceName ""
           PartitionKey = [], Hash
           Indexes = []
+          UniqueKeys = []
```

Comment:
Should this be a set or dictionary (do we need to worry about uniqueness of paths or something)?

Reply:
That's a great point -- duplicate paths would likely be illegal.